### PR TITLE
v2_compressed helpers

### PR DESCRIPTION
**Pull request by joshblum**
Helpers to enable stubbing of links on the server

### CODE REVIEW

File: src/util.iced [hunk 1]
```diff
@@ -11,6 +11,16 @@ exports.json_secure_compare = json_secure_compare = (a,b) ->
 
 ##-----------------------------------------------------------------------
 
+exports.v2_sig_type_from_sig_type = v2_sig_type_from_sig_type = (type) ->
+    # parse out the v2 sig type from a string. i.e. "team.settings -> 46
```
Comment (maxtaco) [inline src/util.iced]: spacing is wrong here

Comment (maxtaco) [reply src/util.iced]: please set your editor to use only 2 spaces in ICS

Comment (joshblum) [reply src/util.iced]: weird i thought it was set..

**Comment by joshblum** [comment]
@maxtaco can you publish the new version?

**Pull request merged**